utilities/calculateFinalPot.ts

```diff
@@ -0,0 +1,36 @@
+import GameSituation from "../models/GameSituation";
+
+export default function calculateFinalPot(
+  gameSituation: GameSituation
+): number {
+  const preflopPot: number[] = new Array(gameSituation.playerLength).fill(0);
```

Comment:
"preflopPot" sounds the pot size that is accumulated chips from all players.

```suggestion
  const lastPlayerBetSizesOnPreflop: number[] = new Array(gameSituation.playerLength).fill(0);
```